tools/publish.sh

```diff
@@ -76,6 +75,7 @@ if ! aws s3api get-bucket-policy --bucket $1 --region $REGION| grep 'Statement'
         read -p "Do you wish to proceed? (yes or no) " yn
         case $yn in
             [Yy]* ) echo "Proceeding..."
+                read -p "Enter the AWS Account ID that will be used to publish to Serverless Application Repository (limits SAR access to applications on the specified account: " account
```

Comment:
Should we validate that account is non empty?  I can see users pressing enter and missing this.

Reply:
true, let me add that